util/SerializeEmpire.cpp

```diff
@@ -69,7 +69,8 @@ void ProductionQueue::Element::serialize(Archive& ar, const unsigned int version
         & BOOST_SERIALIZATION_NVP(turns_left_to_next_item)
         & BOOST_SERIALIZATION_NVP(turns_left_to_completion)
         & BOOST_SERIALIZATION_NVP(rally_point_id)
-        & BOOST_SERIALIZATION_NVP(paused);
+        & BOOST_SERIALIZATION_NVP(paused)
+        & BOOST_SERIALIZATION_NVP(allowed_imperial_stockpile_use);
```

Comment:
use `BOOST_CLASS_VERSION` to preserve backwards save compatibility.

Reply:
Not sure how I do that. As I couldnt find any BOOST_CLASS_VERSION for Empire or Production::Element.
Should I just add a BOOST_CLASS_VERSION(Production::Element, 1) in SerializeEmpire?

Reply:
Not in SerializeEmpire, but otherwise yes. See other uses of it around the code.